src/styles/card.scss

```
@@ -0,0 +1,98 @@
+.card {
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  padding: 32px 16px 16px;
+  width: 200px;
+  font-family: $font-family;
+  background-color: $bg-color;
+
+  &__monitor {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin-inline: 3px;
+    margin-bottom: 40px;
+  }
+
+  &__name {
+    margin-bottom: 4px;
+    color: $main-text-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    margin-top: 0;
+    margin-bottom: 16px;
+    color: $secondary-text-color;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__reviews {
+    display: flex;
+    margin: 0;
+    margin-bottom: 24px;
+    flex-direction: column;
+    justify-content: flex-end;
+    color: $main-text-color;
+    text-align: right;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 166px;
```

No need to hardcode width
```suggestion
    width: 100%;
```